src/pages/events/dialogs/new.tsx

```diff
@@ -308,23 +319,23 @@ export const EventNewIncidentDialog: React.FC<EventNewIncidentDialogProps> = ({
             <option value={IncidentOutcome.Disabled}>Disabled</option>
           </Select>
         </label>
-        <label>
-          <p className="mt-4">Associated Rules</p>
-          <div className="flex mt-2 gap-2">
-            {recentRules?.map((rule) => (
-              <Button
-                className={twMerge(
-                  "text-emerald-400 font-mono",
-                  incident.rules.some((r) => r.rule === rule.rule)
-                    ? "bg-emerald-600 text-zinc-50"
-                    : ""
-                )}
-                onClick={() => onAddRule(rule)}
-                key={rule.rule}
-              >
-                {rule.rule}
-              </Button>
-            ))}
+        <p className="mt-4">Associated Rules</p>
+        <div className="flex mt-2 gap-2">
+          {recentRules?.map((rule) => (
+            <Button
+              className={twMerge(
+                "text-emerald-400 font-mono",
+                incident.rules.some((r) => r.rule === rule.rule)
+                  ? "bg-emerald-600 text-zinc-50"
+                  : ""
+              )}
+              onClick={() => onToggleRule(rule)}
+              key={rule.rule}
+            >
+              {rule.rule}
+            </Button>
+          ))}
+          <label>
```

Comment:
Moving the label to only encapsulate the select, instead of the buttons as well, fixes a minor bug when clicking on the select box
Current behaviour: Initial State
![image](https://github.com/brenapp/referee.fyi/assets/34739837/a4857771-a898-48f7-a58d-789c069e40c3)
On mouse click down (element active)
![image](https://github.com/brenapp/referee.fyi/assets/34739837/c176cbf4-d4e2-4a9e-900c-c0bb4f178786)
On mouse click up (element no longer active)
![image](https://github.com/brenapp/referee.fyi/assets/34739837/e7bf6b2d-810d-4f1f-9903-34c0c52e9368)